lib/files/schema/openapi.json

```diff
@@ -1,1419 +1,2465 @@
-								],
-								"message": "The resources are retrived successfuly."
-							}
-						}
-					}
-				}
-			},
-			"Started": {
-				"description": "A task is started",
-				"headers": {
-					"Retry-After": {
-						"schema": {
-							"type": "integer"
-						}
-					}
-				},
-				"content": {
-					"application/json": {
-						"schema": {
-							"type": "object",
-							"required": [
-								"id", "status_url"
-							],
-							"properties": {
-								"id": {
-									"type": "string",
-									"description": "Id of the newly created resource.",
-									"format": "uuid"
-								},
-								"message": {
-									"type": "string",
-									"description": "A human readble message."
-								},
-								"status_url": {
-									"type": "string",
-									"description": "The URL the to check the status of the running task."
-								}
-							}
-						},
-						"example": {
-							"id": "123e4567-e89b-12d3-a456-426655440000",
-							"message": "The task is started. Use status_url to monitor the progress of the run.",
-							"status_url": "api.pollination.cloud/123e4567-e89b-12d3-a456-426655440000/status"
-						}
-					}
-				}
-			},
-			"Succeed": {
-				"description": "Success"
-			},
-			"Unauthorized": {
-				"description": "Unauthorized.",
-				"content": {
-					"application/json": {
-						"schema": {
-							"$ref": "#/components/schemas/ErrorResponse"
-						}
-					}
-				}
-			},
-			"Unsupported": {
-				"description": "Unsupported media type content.",
-				"content": {
-					"application/json": {
-						"schema": {
-							"$ref": "#/components/schemas/ErrorResponse"
-						}
-					}
-				}
-			}
-		},
-		"parameters": {
-			"resourceId": {
-				"name": "id",
-				"in": "path",
-				"description": "The resource identifier string",
-				"required": true,
-				"schema": {
-					"type": "string",
-					"format": "uuid"
-				}
-			}
-		},
-		"headers": {
-			"ExpiresAfter": {
-				"description": "date in UTC when token expires",
-				"schema": {
-					"type": "string",
-					"format": "date-time"
-				}
-			}
-		},
-		"requestBodies": {
-			"Model": {
-				"content": {
-					"application/json": {
-						"schema": {
-							"oneOf": [
-								{
-									"$ref": "#/components/schemas/Model"
-								}, {
-									"$ref": "#/components/schemas/SpaceBySpaceModel"
-								}, {
-									"$ref": "#/components/schemas/FaceByFaceModel"
-								}
-							]
-						}
-					}
-				},
-				"description": "Pollination Model",
-				"required": true
-			},
-			"SensorGrid": {
-				"content": {
-					"application/json": {
-						"schema": {
-							"$ref": "#/components/schemas/SensorGrid"
-						}
-					}
-				},
-				"description": "SensorGrid",
-				"required": true
-			}
-		},
-		"securitySchemes": {
-			"JWT": {
-				"description": "JSON Web Token (JWT).\nUsage format: `Bearer <JWT>`\n",
-				"type": "http",
-				"scheme": "bearer",
-				"bearerFormat": "JWT"
-			}
-		}
-	}
+  "title": "Model",
```

Comment:
This is up to date with https://github.com/ladybug-tools-in2/energy-model-schema ?

Reply:
@macumber  yes it is